src/components/IstioWizards/Slider/BootstrapSlider.tsx

```diff
@@ -45,8 +48,13 @@ class BootstrapSlider extends React.Component<Props> {
       this.props.onSlide(value);
       this.slider.setValue(value);
     };
+    const onSlideStop = value => {
+      value = value >= this.props.max ? this.props.max : value;
```

Comment:
Since you've used lodash somewhere else: `_.max([this.props.max, value])` sounds better to me, wdyt?

Reply:
+1, may as well use lodash as we already import it. It's more clear. But, actually we want `_.min`.